bin/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+if [ -f "${DOCKER_CERT_PATH}/ca.pem" ]
```

Comment:
I'm a little confused as to why we wouldn't be doing this in the preStart rather than in a wrapper around ContainerPilot?